test/simple.mocha.coffee

```diff
@@ -1,10 +1,14 @@
 { expect } = require("chai")
 
-{simpleLocator} = require('../build/simple')
+# {SimpleLocatorParameters} = require('../src/simple')
+{simpleLocatorFactory} = require('../src/simple')
 
-describe 'Simple locator', ->
+describe.only 'Simple locator', ->
   describe 'shortcut function', ->
-    locator = simpleLocator("localhost:8080")
+    params = SimpleLocatorParameters({
```

Comment:
`SimpleLocatorParameters` is undefined. This type only exists in typescript and not in coffeescript. You should just do a literal object declaration